### DISCUSSION TITLE
bizarre auto deletes of files from excalidraw plugin when using google drive

### DISCUSSION THREAD

**Discussion by Fierce-Ninja** [zsviczian/obsidian-excalidraw-plugin#197] [Q&A]
UPDATE: Please ignore the below topic.  This has become an urgent issue.  I have posted a new issue under the Issues tab for this same topic with more clear description.  Hope someone responds!  Otherwise it is kinda deal breaker.
-------------------------------
Hi I use google drive folder as my vault with google drive desktop app on windows 10.  It works flawlessly with all my obsidian folders until I started using Excalidraw.  For some reason, google drive or Excalidraw plugin or Obsidian (I don't know who) keeps deleting my files, ONLY in excalidraw folder of the vault.  Rest of my files in various folders in the same vault for Obsidian are intact and nothing happens to them.  If it had something to do with google drive, I would have expected such behavior with other files in non-excalidraw folders.  But that doesn't happen.  It is only in Excalidraw folder.  So my rough guess is there is something weird happening in Obsidian or Excalidraw plugin.  Any thoughts on this?

**Comment by zsviczian** [comment]
resolved with #200
